### PR TITLE
tidb-in-kubernetes: add one faq item

### PR DESCRIPTION
**Pull request by xiaojingchen**
### What is changed, added or deleted? 
Add one FAQ item.

### Which version does your change affect? 
dev/v3.0

### CODE REVIEW

File: dev/tidb-in-kubernetes/faq.md [hunk 1]
```diff
@@ -78,3 +78,23 @@ TiDB Operator 尚不支持自动编排 TiSpark。
...
+
+## 部署 TiDB 集群时调度失败是什么原因？
+
+TiDB Operator 调度 Pod 失败的原因可能有三种情况：
+
+* 资源不足的问题，这时 Pod 一直阻塞在 `Pending` 状态，这块我们在集群故障诊断中有详细说明，请参考：[集群故障诊断](/dev/tidb-in-kubernetes/troubleshoot.md)
+
+* 部分 Node 被打了 `taint`，导致 Pod 无法调度到对应的 Node 上。请参考：[taint & toleration](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/)
+
+* 调度错误，这时 Pod 一直阻塞在 `ContainerCreating` 状态。这种情况发生时请检查 Kubernetes 集群中是否部署过多个 TiDB Operator。重复的 TiDB Operator 自定义调度器的存在，会导致同一个 Pod 的调度周期不同阶段会分别被不同的调度器处理，从而产生冲突。
```
Comment (DanielZhangQD) [inline dev/tidb-in-kubernetes/faq.md]: For TiKV and PD, the Pod may be pending due to HA policy.
Maybe the troubleshoot.md should be updated to cover this case.

File: dev/tidb-in-kubernetes/faq.md [hunk 2]
```diff
@@ -78,3 +78,23 @@ TiDB Operator 尚不支持自动编排 TiSpark。
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl get deployment --all-namespaces |grep tidb-scheduler
+    ```
+
+    执行上面命令如果有两条及以上记录，就说明 kubernetes 集群存在重复的 TiDB Operator，请根据实际情况清楚多余的 TiDB Operator。
```
Comment (DanielZhangQD) [inline dev/tidb-in-kubernetes/faq.md]: '清楚' -> '删除'？

File: dev/tidb-in-kubernetes/faq.md [hunk 3]
```diff
@@ -78,3 +78,21 @@ TiDB Operator 尚不支持自动编排 TiSpark。
...
+
+* 调度错误，导致 Pod 一直阻塞在 `ContainerCreating` 状态。这种情况发生时请检查 Kubernetes 集群中是否部署过多个 TiDB Operator。重复的 TiDB Operator 自定义调度器的存在，会导致同一个 Pod 的调度周期不同阶段会分别被不同的调度器处理，从而产生冲突。
+
+执行以下命令，如果有两条及以上记录，就说明 Kubernetes 集群中存在重复的 TiDB Operator，请根据实际情况删除多余的 TiDB Operator。
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl get deployment --all-namespaces |grep tidb-scheduler
+```
```
Comment (DanielZhangQD) [inline dev/tidb-in-kubernetes/faq.md]: This block follows the third case and should be indented, right?

Comment (lilin90) [reply dev/tidb-in-kubernetes/faq.md]: Yes. I'll update it.

**Comment by lilin90** [comment]
@DanielZhangQD PTAL~ Thanks!

File: dev/tidb-in-kubernetes/faq.md (hunk 1 above)
Comment (lilin90) [reply dev/tidb-in-kubernetes/faq.md]: @anotherrachel FYI. Daniel will update `troubleshoot.md` in a new PR.

**Pull request merged**